[2024-10-16T05:02:34.090+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-16T05:02:34.132+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: eia_data_pipeline.call_stored_procedure scheduled__2024-10-15T00:00:00+00:00 [queued]>
[2024-10-16T05:02:34.168+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: eia_data_pipeline.call_stored_procedure scheduled__2024-10-15T00:00:00+00:00 [queued]>
[2024-10-16T05:02:34.171+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-16T05:02:34.213+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): call_stored_procedure> on 2024-10-15 00:00:00+00:00
[2024-10-16T05:02:34.273+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-16T05:02:34.280+0000] {standard_task_runner.py:72} INFO - Started process 115 to run task
[2024-10-16T05:02:34.276+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'eia_data_pipeline', 'call_stored_procedure', 'scheduled__2024-10-15T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/new_dag.py', '--cfg-path', '/tmp/tmpvih4_w7b']
[2024-10-16T05:02:34.283+0000] {standard_task_runner.py:105} INFO - Job 78: Subtask call_stored_procedure
[2024-10-16T05:02:34.412+0000] {task_command.py:467} INFO - Running <TaskInstance: eia_data_pipeline.call_stored_procedure scheduled__2024-10-15T00:00:00+00:00 [running]> on host 7aa864b16460
[2024-10-16T05:02:34.742+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='eia_data_pipeline' AIRFLOW_CTX_TASK_ID='call_stored_procedure' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-15T00:00:00+00:00'
[2024-10-16T05:02:34.746+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-16T05:02:34.748+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-16T05:02:34.751+0000] {logging_mixin.py:190} INFO - Current task name:call_stored_procedure state:running start_date:2024-10-16 05:02:34.133379+00:00
[2024-10-16T05:02:34.753+0000] {logging_mixin.py:190} INFO - Dag name:eia_data_pipeline and current dag run status:running
[2024-10-16T05:02:34.756+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-16T05:02:41.833+0000] {logging_mixin.py:190} INFO - Stored procedure executed successfully.
[2024-10-16T05:02:41.839+0000] {logging_mixin.py:190} INFO - MySQL connection is closed
[2024-10-16T05:02:41.842+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-16T05:02:41.865+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-16T05:02:41.867+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=eia_data_pipeline, task_id=call_stored_procedure, run_id=scheduled__2024-10-15T00:00:00+00:00, execution_date=20241015T000000, start_date=20241016T050234, end_date=20241016T050241
[2024-10-16T05:02:41.890+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-16T05:02:41.891+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-16T05:02:41.893+0000] {logging_mixin.py:190} INFO - Dag name:eia_data_pipeline queued_at:2024-10-16 05:01:21.610292+00:00
[2024-10-16T05:02:41.895+0000] {logging_mixin.py:190} INFO - Task hostname:7aa864b16460 operator:PythonOperator
[2024-10-16T05:02:41.945+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-16T05:02:41.990+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T05:02:41.995+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
